summit/auxtel-ill-control/setup_dev_fiberspectrograph.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Source this file when starting the container to set it up
+echo "Running setup for environment for FiberSpectrograph"
+source /home/saluser/.setup_salobj.sh
+
+setup ts_config_ocs -t current
+setup ts_idl -t current
+setup ts_sal -t current
+setup ts_salobj -t current
+setup ts_xml -t current
+
+echo "# Running custom setups of repos"
+setup -j ts_config_ocs -r ~/develop/ts_config_ocs
+setup -j ts_fiberspectrograph -r ~/develop/ts_FiberSpectrograph
+setup -j ts_utils -r ~/develop/ts_utils
+
+echo '#Installing dependencies and declaring OSPL_URI'
```

Comment:
Please add a space after the `#` because you do that everywhere else in this file.

Reply:
Fixed.